assets/stylesheets/_components.scss

```diff
@@ -178,6 +178,7 @@
 @import 'components/payment-logo/style';
 @import 'components/plans/plan-icon/style';
 @import 'components/plans/plan-price/style';
+@import 'components/plans/plans-skip-button/style';
```

Comment:
Been wondering the same. Then again, there's also `components/plans/plan-icon` and `components/plans/plan-price` (which are singular, granted). Inclined to leave as is, so that e.g. the `className` can also remain `plans-skip-button` and not just `skip-button`.